vendor.conf

```diff
@@ -26,7 +26,7 @@ github.com/imdario/mergo v0.3.6
 golang.org/x/sync 1d60e4601c6fd243af51cc01ddf169918a5407ca
 
 # buildkit
-github.com/moby/buildkit e57eed420c7573ae44875be98fa877175b4677a1
+github.com/moby/buildkit 95e0348f571a0e500a9b6911992c92f877b44ff1 git://github.com/tonistiigi/buildkit.git
```

Comment:
https://github.com/moby/buildkit/pull/560 was merged; so this can likely be "un-forked"